src/lib/components/forms/picture-input.svelte

```diff
@@ -32,22 +35,24 @@
     style="display:none"
     type="file"
     {name}
+    {value}
+    {required}
     accept="image/*"
     on:change={(e) => onFileSelected(e)}
     bind:this={fileInput}
   />
   <button
     type="button"
     aria-label="Upload image"
-    class="relative flex h-[200px] w-[200px] items-center justify-center rounded-md bg-muted-red-400 text-center"
+    class="hover= relative flex h-[200px] w-[200px] items-center justify-center rounded-md bg-muted-red-700 text-center hover:bg-muted-red-500"
```

Comment:
```suggestion
    class="relative flex h-[200px] w-[200px] items-center justify-center rounded-md bg-muted-red-700 text-center hover:bg-muted-red-500"
```

I think this was leftover